examples/emulator_examples/add_trial_events.py

```diff
@@ -0,0 +1,58 @@
+import secrets
```

Comment:
I had random but the codacy (github's code quality bot) warned that it's insecure. Although we don't care about his within our context, I added `secrets` for the sake of no unnecessary warnings. Just to be 'clean', I am more inclined to committing `random` -> `secrets` in all examples that contain it, if you agree.

Reply:
Maybe we should change random -> secrets in a future PR in order to solve the Codacy security error:

> Standard pseudo-random generators are not suitable for security/cryptographic purposes.

Reply:
It's up to the pybpod developers, but IMO if we aren't doing crypto then it's misleading (and probably more computationaly intensive, but not the issue here) to use `secrest` rather than `random`. I would indeed prefer to leave them all as `random`

Reply:
Will fix.